### ISSUE TITLE
[vNext] Add buttons from Examples\UX\Prefabs to UX\Prefabs\Buttons

### ISSUE THREAD

**Issue by davidkline-ms** [Microsoft/MixedRealityToolkit-Unity#2010]
## Overview
The HoloToolkit-Examples package has a set of highly useful buttons in it's UX\Prefabs folder that should actually be part of the toolkit itself.

The following should be moved from examples into the toolkit for greater visibility and use:
- Button
- ToggleButton
- Checkbox
- Radial
- Toggle
- Slider (not technically a button :) )

These are all featured in the InteractiveButtonComponents scene, which will need to be updated as well.

**Comment by cre8ivepark** [comment]
@killerantz can provide insights here😄

**Comment by killerantz** [comment]
I made some improvements to the InteractiveElements components in a side project, but the changes are breaking. There are some nicer features in the new work which I could bring over, but after speaking with @cre8ivepark I started looking at combining InteractiveElements with InteractableObjects in the examples and create a new set of components. This work is in progress as outlined here #1625. I have a rough prototype and can move what I have to a public branch, just let me know.

The work on it has slowed down over the past few weeks because of work conflicts so I don't have a timeline as of now.
